packages/presentation/lib/library/dimens/dimens.dart

```diff
@@ -0,0 +1,146 @@
+class Dimens {
+  Dimens._();
+
+  ///height
+  static const double height2 = 2;
```

Comment:
imho, instead of all consts type (height + padding + width + border + size) u can use just one - `dimen`